.gitignore

```diff
@@ -13,5 +13,8 @@ src/main/resources/docs/
 *.iml
 bin/
 
+/data/
+
 /text-ui-test/ACTUAL.txt
 text-ui-test/EXPECTED-UNIX.TXT
+data/
```

Comment:
`/data/` and `data/` behave differently in `.gitignore` files. 

You might risk ignoring any `data` folders in the entire project if you use `data/`. 

Only `/data/` is enough for our case, since we are ignoring the folder only in the present directory, and not another instance of `data` like `SUBFOLDER/data`.